operator/src/main/java/org/keycloak/operator/v2alpha1/crds/KeycloakSpec.java

```diff
@@ -29,7 +29,7 @@
 
     private int instances = 1;
     private String image;
-    private Map<String, String> serverConfiguration;
+    private Map<String, ValueOrSecret> serverConfiguration;
```

Comment:
The expected `yaml` signature (to be aligned with other standard kubernetes resources) should be:
```yaml
- name: MYKEY
  value: MYVALUE
```
and not:
```yaml
- MYKEY:
    value: MYVALUE
```

This encoding will look more aligned to [env-vars](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#define-an-environment-variable-for-a-container), [secrets, volume mounts](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod) and more.

Reply:
👍